independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/ClientBuilderImpl.java

```diff
@@ -114,8 +114,8 @@ public ClientBuilder trustStore(KeyStore trustStore, char[] password) {
 
     @Override
     public ClientBuilder hostnameVerifier(HostnameVerifier verifier) {
-        this.hostnameVerifier = verifier;
-        return this;
+        // TODO
+        throw new RuntimeException("Specifying HostnameVerifier is not supported at the moment");
```

Comment:
This doesn't look right to me

Reply:
I first thought the same, but this is exactly was the existing code does for the same unsupported feature of custom `SSLContext` for the reactive client.

https://github.com/quarkusio/quarkus/blob/3.0.1.Final/independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/ClientBuilderImpl.java#L89


This is existing code from the same class:
````java
    @Override
    public ClientBuilder sslContext(SSLContext sslContext) {
        // TODO
        throw new RuntimeException("Specifying SSLContext is not supported at the moment");
    }
````

From my perspective failing early here is actually better than the current code, that makes it look like the hostname verifier is applied, but then never actually used by the reactive client.

Reply:
How about throwing an `UnsupportedOperationException` instead?

Reply:
That fits better. Thanks. Switched both for `UnsupportedOperationException` to keep it consistent.

Fixed in https://github.com/quarkusio/quarkus/pull/32932/commits/7f7d717b54cbaf8f03f00c33f1b3aa987e54cbaf